docs/conf.py

```diff
@@ -19,7 +16,7 @@
 project = u'astor'
 copyright = u'2013-%s, Berker Peksag' % time.strftime('%Y')
 
-version = release = setuputils.read('astor/VERSION')
+version = release = astor.__version__
```

Comment:
I usually try to avoid importing the library inside `conf.py`. Maybe we can do `version = release = '0.8.1'` here for now and then start to use `bumpversion` in another PR?

Reply:
> I usually try to avoid importing the library inside `conf.py`. Maybe we can do `version = release = '0.8.1'` here for now and then start to use `bumpversion` in another PR?

As you wish :)
Resolved in 33b5c81